code/drasil-printers/Language/Drasil/Printing/Import/Document.hs

```diff
@@ -101,7 +101,7 @@ layUnlabelled  _ (Bib bib)              = T.Bib $ map layCite bib
 
 -- | For importing a bibliography.
 layCite :: Citation -> P.Citation
-layCite c = P.Cite (c ^. citeID) (c ^. citeKind) (map layField (c ^. getFields))
+layCite c = P.Cite (showUID c) (c ^. citeKind) (map layField (c ^. getFields))
```

Comment:
This feels wrong. Either because the `UID` is used for something real, or because it's assumed to be a `String`. Both are wrong.

Reply:
I found that `Cite` from `Language.Drasil.Printing.Citation` is made using an `EntryID` rather than a `UID` which is just a direct synonym for `String`. Should it take a `UID` instead?

Reply:
So I think the idea is that an `EntryID` might actually be different. In fact, I think it's a hack that we're using the UID for the EntryID. The EntryID will be user-visible, at least inside the sources. UIDs, other than in debugging (or the dependency graph) really should be invisible.

But let's not over-complicate: keep the hack, make an issue, it can be fixed after.